frontend-wasm2/src/code_translator/mod.rs

```diff
@@ -139,15 +131,15 @@ pub fn translate_operator(
                     state.push1(builder.ins().select(cond, arg1, arg2, span)?);
                 }
                 wasmparser::ValType::I64 => {
-                    todo!()
-                    // let cond = builder.ins().neq_imm(cond, Immediate::I64(0), span);
-                    // state.push1(builder.ins().select(cond, arg1, arg2, span)?);
+                    let imm = builder.ins().imm(Immediate::I64(0), span);
+                    let cond = builder.ins().neq(cond, imm, span)?;
+                    state.push1(builder.ins().select(cond, arg1, arg2, span)?);
                 }
                 ty => panic!("unsupported value type for 'select': {ty}"),
             };
         }
         Operator::Unreachable => {
-            builder.ins().unreachable(span);
+            builder.ins().unreachable(span)?;
```

Comment:
*Note:* I think I already updated this, but just in case, the need for dedicated instruction builder, i.e. the reason why we have the `.ins()` builder method, is gone. Now any implementation of `OpBuilder` or `Rewriter` can be used to build ops, just by bringing the dialect-specific builder trait into scope, e.g. `UndefinedBehaviorOps` in the case of `unreachable`.

I believe I already updated the codebase to use the new style though, so probably this is just something that will go away after a rebase.

Reply:
Yep, it's gone after rebase. I removed `.ins()` in the new code.